spec/services/cfe/submission_manager_spec.rb

```diff
@@ -6,13 +6,14 @@ module CFE
     #   VCR.configure { |c| c.ignore_hosts 'localhost' }
     # end
 
-    let(:application) { create :legal_aid_application, :with_everything, :at_provider_submitted }
+    let(:application) { create :legal_aid_application, :with_everything, :with_vehicle, :at_provider_submitted, populate_vehicle: true }
```

Comment:
I think we need to stop relying on complicated FactoryBot trait constructions. They are very difficult to debug when problems occur.

This is much simpler and a lot less likely to come back and bite us later:
```ruby
let(:vehicle) { create :vehicle, :populated }
let(:application) { create :legal_aid_application, :with_everything, :at_provider_submitted, vehicle: vehicle
```